R/drive_rm.R

```diff
@@ -32,6 +33,5 @@ delete_one <- function(id) {
     endpoint = "drive.files.delete",
     params = list(fileId = id)
   )
-  response <- make_request(request)
-  process_response(response)
+  do_request(request)
```

Comment:
oh I forgot that I had changed this before! I'll put it back to `make_request()` and `process_response()`.